src/widgets/about-community/ui/about-community.module.scss

```diff
@@ -0,0 +1,36 @@
+.about-community {
+  gap: 176px;
+
+  .about-info {
+    min-width: 640px;
+
+    @include media-laptop {
+      gap: 20px;
+      min-width: 400px;
+    }
+
+    @include media-tablet {
+      min-width: auto;
+    }
+  }
+
+  img {
```

Comment:
Maybe we could use a more unique class selector here than the image tag?

Reply:
Is `+ img` will be more specific? Because we use modules and in this module we have only one `img`